charts/shoot-core/components/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: kube-system
   labels:
     gardener.cloud/role: system-component
+    # TODO (ialidzhikov): remove in a future version
```

Comment:
Why not now?

Reply:
Because there is currently a netpol that is using the deprecated label and this netpol is adapted to the new label key in this PR.

Reply:
I see, thanks for the explanation!